common/display/displayplanemanager.cpp

```diff
@@ -498,7 +498,8 @@ void DisplayPlaneManager::ValidateCursorLayer(
       }
 
       last_plane = GetLastUsedOverlay(composition);
-      is_video = last_plane->IsVideoPlane();
+      if (last_plane)
```

Comment:
While I understand the check, it doesn't make much sense that we are ending up with no last overlay unless for some reason we are just rendering cursor layer and nothing else. So, I guess something is broken earlier in the system, @renchenglei You mentioned about some under runs in the email, what's the fix for it ?

Reply:
@kalyankondapally this is not for "under runs", it is fix for https://jira01.devtools.intel.com/browse/OAM-67193